### PR TITLE
Add details about how to use this project

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi @ms-darianbenito, @javierburger, @hrocca, @mcappato, @elsupergomez, and team!

You have already used this example project to configure new ones.

I think that this information was missed, I was not sure where to document it, but here is better than in no place, right?

Could you review this? is something missed? is it enough clear? is my English acceptable?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +57,67 @@ If you prefer to run these commands without docker, you can read [Dockerfile](./
...
+- Case sensitive _search and replace_ all occurrences of `hello-microservice` to the desired name.
+
+- Replace the IDs in the .sln file.
+
+  We have `{FC79D827-86F3-4F93-8064-C4927957A1D2}` for the main project, `{646F92A1-8B19-43E0-BED1-8A56B6FB9352}` for the test project, `{9BE75F53-F47E-4B35-9560-AA039EBC5B1C}` for the solution files and `{A6DDDBE4-738F-4679-809B-D7786BD5E7E5}` for the solution itself. It is possible generate new IDs using any tool, for example [Online GUID / UUID Generator](https://www.guidgenerator.com/)
+
+- Ensure that the file `.doppler-ci` is a _symbolic link_ to `Jenkinsfile`
+
+  - In Linux or Mac: `ln -s Jenkinsfile .doppler-ci`
+  - In Windows (with git bash): `export MSYS=winsymlinks:nativestrict; ln -s Jenkinsfile .doppler-ci`
```
Comment (ms-darianbenito) [inline README.md:86]:
```suggestion
  - In Windows (with git bash as administrator): `export MSYS=winsymlinks:nativestrict; ln -s Jenkinsfile .doppler-ci`
```

**Comment by ms-darianbenito** [comment]
Maybe, we can include a small suggestion to remove the following lines from the `Dockerfile` if the current project does not need a SQL server connection:
image

**Comment by ms-darianbenito** [comment]
👍

**Comment by andresmoschini** [comment]
> Maybe, we can include a small suggestion to remove the following lines from the `Dockerfile` if the current project does not need a SQL server connection:
> image

In my opinion, we should simply copy those lines. Why worry about it?

**Comment by ms-darianbenito** [comment]
> > Maybe, we can include a small suggestion to remove the following lines from the `Dockerfile` if the current project does not need a SQL server connection:
> > image
> 
> In my opinion, we should simply copy those lines. Why worry about it?

No problem with these lines, we can keep them

File: README.md [hunk 2]
```diff
@@ -57,3 +57,67 @@ If you prefer to run these commands without docker, you can read [Dockerfile](./
...
+![github-main-protection-rules](./docs/github-main-protection-rules.png)
+
+### Configure Docker Hub Webhooks
+
+At this point, the CI process already generated a Docker Hub repository and we should configure the WebHooks to enable the auto-redeploy in our environments.
+
+Open the Webhooks configuration page of this new repository (this is the URL for hello-microservice repository: <https://hub.docker.com/repository/docker/dopplerdock/hello-microservice/webhooks>) and create the webhooks for production and test environments:
+
+- **cd-helper-production** `https://apis.fromdoppler.com/cd-helper/hooks/{{REEMPLACE-THE-SECRET-HERE}}/`
+- **cd-helper-test** `https://apisqa.fromdoppler.net/cd-helper/hooks/{{REEMPLACE-THE-SECRET-HERE}}/`
```
Comment (javierburger) [inline README.md:117]: could you explain how to obtain the secret?

**Comment by elsupergomez** [comment]
> Maybe, we can include a small suggestion to remove the following lines from the `Dockerfile` if the current project does not need a SQL server connection:
> image

If not requiere connection to old versions of SQL Server

**Review by elsupergomez — COMMENTED** [review]
👍 

Could be nice have a `script` to do all task to use this repo as a template. Just an idea, maybe for the scope of another PR

**Comment by andresmoschini** [comment]
> Could be nice have a `script` to do all task to use this repo as a template. Just an idea, maybe for the scope of another PR

@hrocca  purposes something like yeoman

File: README.md (hunk 2 above)
Comment (andresmoschini) [reply README.md:117]: Nope, you have to ask :P

**Pull request merged**